src/main/scala/net/snowflake/spark/snowflake/io/CloudStorageOperations.scala

```diff
@@ -579,6 +582,7 @@ sealed trait CloudStorage {
              | partition ID:$partitionID $fileName
              | attemptNumber=$attemptNumber
              | backoffTime=${Utils.getTimeString(sleepTime)}
+             | useExponentialBackoff=$useExponentialBackoff
```

Comment:
This log message with backOffTIme is confusing if we're not actually backing off.
Remove the backoffTime line from this message from here and add it to just above the Thread.sleep below.